modules/sno-du-tuning-the-performance-patch.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-tuning-the-performance-patch_{context}"]
+= Tuning the performance patch
+
+
```

Comment:
@imiller0 This is one of the new topics. Please provide an intro sentence here - basically, why they are using this procedure.

Reply:
added.